openpype/hosts/maya/startup/userSetup.py

```diff
@@ -4,10 +4,16 @@
 from openpype.hosts.maya.api import MayaHost
 from maya import cmds
 
+# MAYA_RESOURCES enviornment variable is referenced in default OCIO path but
+# it's not part of the environment. Patching this so it works as expected.
+if "MAYA_RESOURCES" not in os.environ:
+    os.environ["MAYA_RESOURCES"] = os.path.join(
+        os.environ["MAYA_LOCATION"], "resources"
+    ).replace("\\", "/")
+
```

Comment:
```suggestion
```

Let's remove this. See [comment here](https://github.com/ynput/OpenPype/pull/4340#discussion_r1073969460) which apparently doesn't show on the PR :)